operator/pkg/controlplane/control_plane.go

```diff
@@ -77,7 +77,7 @@ func NewIstioOperator(installSpec *v1alpha1.IstioOperatorSpec, translator *trans
 	return out, nil
 }
 
-func orderedKeys(m map[string]*v1alpha1.ExternalComponentSpec) []string {
+func OrderedKeys(m map[string]*v1alpha1.ExternalComponentSpec) []string {
```

Comment:
Why does this need to change? You can test unexported functions because the test is in the same package as the code